IntegrationTests.Common/TestDomain/Northwind/NorthwindDataContext.cs

```diff
@@ -3955,8 +3955,14 @@ public partial class Invoices
 
     private System.Nullable<System.DateTime> _OrderDate;
 
+#if NET6_0_OR_GREATER
+    private System.Nullable<System.DateOnly> _RequiredDate;
+
+#else
     private System.Nullable<System.DateTime> _RequiredDate;
 
+#endif
```

Comment:
I just had an idea, not sure if it could work: we could use an alias-using to define an alias for DateOnly and reference that instead of switching each location.
```
#if NET6...
  using DateOnly2 = System.DateOnly;
#else
  using DateOnly2 = System.DateTime;
#elseif

 private System.Nullable<DateOnly2> _RequiredDate;
```